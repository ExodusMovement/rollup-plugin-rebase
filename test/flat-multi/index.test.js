import { bundle, clean, list, read } from "../util"

const root = __dirname

test("Flat Multi", async () => {
  await bundle(root, "index.js", "output/index.js")

  expect(list(root, "output")).resolves.toMatchSnapshot()
  expect(read(root, "output/index.js")).resolves.toMatchSnapshot()

  await clean(root, "output")
})
