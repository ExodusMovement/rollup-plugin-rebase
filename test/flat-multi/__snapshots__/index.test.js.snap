// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flat Multi 1`] = `
"bXipeBrs.png
bsdudxaF.md
index.js"
`;

exports[`Flat Multi 2`] = `
"import content from './bsdudxaF.md';
import image from './bXipeBrs.png';

function index() {
  return content + image
}

export default index;
"
`;
